### PR TITLE
Github Actions for Windows

### PR DESCRIPTION
**Pull request by danrbailey**
Add a new CI job for GitHub Actions that tests the Windows build and delete the AppVeyor one.

For now, it's roughly equivalent to AppVeyor - it uses a random selection of library versions that happen to be available in vcpkg and only tests a subset of VDB (unit tests and vdb print binary).

I suggest we make the transition soon and then continue to submit more PRs to extend this to support additional functionality (python module, other binaries, houdini, etc).

(@e4lam)

### CODE REVIEW

**Comment by e4lam** [comment]
In `install_windows.sh`, do we really need the `vcpkg integrate install`? My reading of the docs on this is that it's only necessary for Visual Studio projects that don't use the cmake toolchain file method.

**Comment by danrbailey** [comment]
> In `install_windows.sh`, do we really need the `vcpkg integrate install`? My reading of the docs on this is that it's only necessary for Visual Studio projects that don't use the cmake toolchain file method.

I think you're right, I just removed it. I also created a new LF service desk ticket to request disabling AppVeyor, I believe it needs re-authenticating so we might as well disable it now.

**Comment by e4lam** [comment]
@danrbailey I just noticed that we also still have a `vcpkg integrate` command listed in README.md at Line 114 in the Windows build section.

**Comment by danrbailey** [comment]
> @danrbailey I just noticed that we also still have a `vcpkg integrate` command listed in README.md at Line 114 in the Windows build section.

Thanks, just updated this!

**Comment by danrbailey** [comment]
@Idclip or @jmlait - would one of you mind approving this so that I can re-introduce windows testing?

File: ci/build_windows.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+set -ex
+
+VCPKG_ROOT="$1"
+
+mkdir build
+cd build
+cmake \
+  -G "Visual Studio 16 2019" -A x64 \
+  -DCMAKE_TOOLCHAIN_FILE="${VCPKG_ROOT}\scripts\buildsystems\vcpkg.cmake" \
+  -DOPENVDB_BUILD_BINARIES=ON \
```
[the remark this answers is not in the document]
Comment (danrbailey) [reply ci/build_windows.sh]: Yeah, we can add these in a subsequent pass, I just wanted to get it enabled in some form...

**Pull request merged**